src/styles/_vars.scss

```diff
@@ -24,11 +24,15 @@ $lu_upset_circle_color: black !default;
 $lu_upset_inactive_opacity: 0.1 !default;
 $lu_upset_stroke: black !default;
 
-$lu_engine_grip_gap: 5px !default;
+$lu_hist_color: #c1c1c1;
 
-$lu_missing_bg: repeating-linear-gradient(-45deg, transparent, transparent 7px, #ddd 7px, #ddd 10px) !default;
+$lu_engine_grip_gap: 3px !default;
```

Comment:
I think there is a corresponding TypeScript variable that also needs to be changed, ~ columnPadding or something like that.

Reply:
Correct. Thanks for the hint. I changed it.